opfunu/name_based/a_func.py

```diff
@@ -69,6 +69,7 @@ class Ackley02(Benchmark):
     parametric = False
 
     modality = False  # Number of ambiguous peaks, unknown # peaks
+    ndim_max = 2
```

Comment:
@firestrand ,

Like I said, that is the way. I can't validate your PR since it is too big. And you should just focus on bugs in each functions instead of the core functions that can affect entire library. The core functions that I wrote it is not 1 or 2 days. It is a year of implementing and improving. Of course, there is some places that it can be optimized, but it is not a bug.